v2.1/set-vars.md

```diff
@@ -40,6 +40,7 @@ The variable name is case insensitive. The value can be a list of one or more it
  `database` | The [current database](sql-name-resolution.html#current-database).  Database in connection string, or empty if not specified | Yes
  `default_transaction_isolation` | The default transaction isolation level for the current session. See [Transaction parameters](transactions.html#transaction-parameters) and [`SET TRANSACTION`](set-transaction.html) for more details. | Settings in connection string, or "`SERIALIZABLE`" if not specified  | Yes
  `default_transaction_read_only` | The default transaction access mode for the current session. If set to `on`, only read operations are allowed in transactions in the current session; if set to `off`, both read and write operations are allowed. See [`SET TRANSACTION`](set-transaction.html) for more details. | `off` | Yes
+ `distsql` | <span class="version-tag">New in v2.1:</span> The query distribution mode for the current session.<br><br>If `auto`, all queries that should be distributed (determined by a heuristic) are distributed, and all other queries are run through the gateway node. **`auto` is recommended.** <br><br>If `on`, any queries that can be distributed are distributed, and all other queries are run through the gateway node.<br><br>If `off`, all queries are run through the local SQL engine. <br><br>If `always`, all queries are distributed, even those that can't be distributed (returns an error). This setting is used for internal testing and is not recommended. | `auto` | Yes
```

Comment:
I would flesh out the language `should` by a bit. Maybe say something like "if `auto`, all queries that CockroachDB believes will take less time to execute if distributed to multiple nodes will be executed". Probably there's a better way to word that, but the important point is that the heuristic is based on whether or not we think it'll be faster to run the query if we distribute it.